metricbeat/module/prometheus/collector/_meta/docs.asciidoc

```diff
@@ -24,6 +24,58 @@ to retrieve the metrics from (`/metrics` by default) can be configured with `met
 -------------------------------------------------------------------------------------
 
 
+[float]
+[role="xpack"]
+=== Histograms and types
+
+beta[]
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+metricbeat.modules:
+- module: prometheus
+  period: 10s
+  hosts: ["localhost:9090"]
+  use_types: true
+  rate_counters: false
+-------------------------------------------------------------------------------------
+
+`use_types` paramater (default: false) enables a different layout for metrics storage, leveraging Elasticsearch
+types, including https://www.elastic.co/guide/en/elasticsearch/reference/current/histogram.html[histograms].
+
+`rate_counters` paramater (default: false) enables calculating a rate out of Prometheus counters. It will stored
```

Comment:
Maybe `It will stored the counter increment since the last collection.` needs some improvement to make more sense?